docs/deployments/apis.md

```diff
@@ -10,6 +10,7 @@ Serve models at scale.
   model: <string>  # path to an exported model (e.g. s3://my-bucket/exported_model)
   model_format: <string>  # model format, must be "tensorflow" or "onnx" (default: "onnx" if model path ends with .onnx, "tensorflow" if model path ends with .zip or is a directory)
   request_handler: <string>  # path to the request handler implementation file, relative to the cortex root
+  tf_signature_key: <string> # The name of the signature def to use for prediction (optional if your model has only one signature def)
```

Comment:
`# name of the signature def to use for prediction (required when your model has more than one signature def)`